### PR TITLE
Q14siX.de.Mathematikum version 1.0.0.0

### PR DESCRIPTION
**Pull request by Q14siX**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/108113)

### CODE REVIEW

File: manifests/q/Q14siX/de/Mathematikum/1.0.0.0/Q14siX.de.Mathematikum.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Q14siX.de.Mathematikum
+PackageVersion: 1.0.0.0
+Installers:
+- InstallerLocale: de-DE
+  Architecture: x86
```
Comment (Trenly) [inline manifests/q/Q14siX/de/Mathematikum/1.0.0.0/Q14siX.de.Mathematikum.installer.yaml:8]:
```suggestion
  Architecture: x64
```

**Review by Trenly — CHANGES_REQUESTED** [review]

**Comment by stephengillie** [comment]
Automatic Validation ended with:
> Installation failed with exit code -1978334957 2023-05-25 10:07:02.130 [CLI ] ShellExecute installer failed: 1633 
> MSI (s) (4C:BC) [10:07:01:972]: Windows Installer installed the product. Product Name: Mathematikum. Product Version: 1.0.0.0. Product Language: 1031. Manufacturer: Q14siX.de. Installation success or error status: 1633.

**Comment by stephengillie** [comment]
Manual Validation failed with:
> Faulting application name: Mathematikum.exe, version: 1.0.0.0, time stamp: 0xe815d0ab  
> Faulting module name: KERNELBASE.dll, version: 10.0.22621.1778, time stamp: 0x11df5af5 Exception code: 0xe0434352 Fault offset: 0x00148462  
> Faulting process id: 0x0x1F48  
> Faulting application start time: 0x0x1D98F70F6653558  
> Faulting application path: C:\Program Files\Q14siX.de\Mathematikum\Mathematikum.exe  
> Faulting module path: C:\Windows\System32\KERNELBASE.dll Report Id: d37e8174-ca36-411e-929b-4ccbe6140865  
> Faulting package full name:   
> Faulting package-relative application ID:  

The package installed normally, but gave the above error when launched.

**Comment by ItzLevvie** [comment]
Close with reason: Stale pull request;

**Pull request closed without merging**